quote_to_image.php

```diff
@@ -16,10 +16,16 @@
 
 // pad naar font file
 putenv('GDFONTPATH=' . realpath('.'));
+
 $font_path = 'LinLibertine_RZah.ttf';
 $font_path_bold = 'LinLibertine_RBah.ttf';
 $creditFont = 'LinLibertine_RZIah.ttf';
 
```

Comment:
The instructions I followed installed the fonts as *.otf
So I need to uncomment the lines below in order to run successfully